### PR TITLE
Rename some of the fields in wallet

### PR DESCRIPTION
**Pull request by jalextowle**

### CODE REVIEW

File: src/elfpy/wallet.py [hunk 1]
```diff
@@ -16,21 +16,23 @@ class Wallet:
 
     # fungible
     address: int
-    base_in_wallet: float
-    lp_in_wallet: float = 0  # they're fungible!
+    base: float
+    lp: float = 0  # pylint: disable=invalid-name
```
Comment (dpaiton) [inline src/elfpy/wallet.py:20]: what if you just made this a valid name? like liquidity?

File: src/elfpy/wallet.py [hunk 2]
```diff
@@ -16,21 +16,23 @@ class Wallet:
 
     # fungible
```
Comment (dpaiton) [inline src/elfpy/wallet.py:17]: Can you add a description of each of these variables in the docstring for this class?

File: src/elfpy/wallet.py (hunk 1 above)
Comment (jalextowle) [reply src/elfpy/wallet.py:20]: Liquidity doesn't really have the same meaning to be honest. The linting directive doesn't really bother me.

**Review by dpaiton — APPROVED** [review]
Change `lp` to `lp_tokens` and then it's good.

**Pull request merged**